mathics/builtin/inout.py

```diff
@@ -65,6 +66,20 @@
 SymbolSubscriptBox = Symbol("SubscriptBox")
 
 
+class ShowSteps(Predefined):
+    """
+    If this variable is set to `True`, the steps in the evaluation are shown.
+    """
+
+    name = "ShowSteps"
```

Comment:
I'm not sure what is the naming convention, but I feel like this should be `$ShowSteps`, and there should be also the function `ShowSteps` (like `TraceBuiltins`).

Reply:
I thought about this as a configuration variable, but using an activating/deactivating function could also work.